src/test/scala/org/broadinstitute/clio/dataaccess/ElasticsearchContainer.scala

```diff
@@ -1,28 +1,56 @@
 package org.broadinstitute.clio.dataaccess
 
+import java.util.concurrent.TimeUnit
+
 import com.dimafeng.testcontainers.{ForAllTestContainer, GenericContainer}
 import org.scalatest.Suite
 import org.testcontainers.containers.wait.Wait
+import org.testcontainers.images.RemoteDockerImage
 
 /**
   * Elasticsearch Test Container
+  *
+  * Setup adapted from https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html
+  *
+  * X-Pack disabled so that logins aren't required.
+  *
+  * Transport and HTTP hosts explicitly wired to local IP addresses so that Elasticsearch's automatic bootstrap checks
+  * are never initiated.
+  * - [[https://www.elastic.co/guide/en/elasticsearch/reference/5.1/docker.html#docker-cli-run-dev-mode]]
+  * - [[https://www.elastic.co/blog/bootstrap_checks_annoying_instead_of_devastating]]
   */
 trait ElasticsearchContainer extends ForAllTestContainer {
   self: Suite =>
 
-  // Adapted from https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html
-  val container =
+  private lazy val elasticsearchImage = {
+    // On some systems, "docker.io/" gets automagically prepended to the image names, causing failures during
+    // `new RemoteDockerImage(image).get()`. If the first attempt fails, it is logged by Testcontainers during this
+    // resolution. If/when the second attempt fails on the modified image name, the latter error is printed during
+    // `val container` below.
+    val image = TestContainers.DockerImages.elasticsearch
```

Comment:
As a workaround, this seems OK to me. It wouldn't be that much harder to make a general version, though.